### PR TITLE
Fixed `docker-compose` file to work with Windows

### PR DESCRIPTION
**Pull request by dwjorgeb**
Added `factomd.conf` file to point to the right daemon containers
Moved `defaultconfig.ini` file to `config/` dir
Updated documentation to reflect the changes
Fixed `Dockerfile` file to point to new config path

### CODE REVIEW

File: Docker.md [hunk 1]
```diff
@@ -61,9 +67,9 @@ Or a node with a set number of miners:
 docker-compose run --rm pegnet go run main.go --log=debug --miners=4
```
Comment (afenrir) [inline Docker.md]: `main.go` is now `pegnet.go` on develop. Please would you change this?

Comment (dwjorgeb) [reply Docker.md]: done :)

Comment (afenrir) [reply Docker.md]: Sorry, my bad, line 60 also references main.go. Would you mind changing that too? Other than that problem, I have tested this and it works, so will merge as soon as that is changed.

Comment (dwjorgeb) [reply Docker.md]: whoops, missed that one. Fixed @afenrir

**Pull request merged**